content/en/docs/guides/opentelemetry-metrics.md

```diff
@@ -0,0 +1,148 @@
+---
+title: OpenTelemetry Metrics
+description: OpenTelemetry Metrics available in gRPC
+---
+
+## Overview
+
+gRPC provides support for an OpenTelemetry plugin that provides metrics that can help you -
+* Troubleshoot your system
+* Iterate on improving system performance
+* Setup continuous monitoring and alerting.
+
+## Background
+
+OpenTelemetry is an observability framework to create and manage telemetry data. gRPC previously provided observability support through OpenCensus which has been [sunsetted](https://opentelemetry.io/blog/2023/sunsetting-opencensus/) in the favor of OpenTelemetry.
+
+## Instruments
+
+The gRPC OpenTelemetry plugin accepts a [MeterProvider](https://opentelemetry.io/docs/specs/otel/metrics/api/#meterprovider) and depends on the [OpenTelemetry API](https://opentelemetry.io/docs/specs/otel/overview/#api) to create a [Meter](https://opentelemetry.io/docs/specs/otel/metrics/api/#get-a-meter) that identifies the gRPC library being used, for example, `grpc-c++` at version `1.57.1`. The following listed instruments are created using this meter. Users should employ the [OpenTelemetry SDK](https://opentelemetry.io/docs/specs/otel/overview/#sdk) to customize the views exported by OpenTelemetry.
+
+More and more gRPC components are being instrumented for observability. Currently, we have the following components instrumented -
+* Per-call (stable, on by default) : Observe RPCs themselves (for example, latency.)
+  * Client Per-Call : Observe a client call
+  * Client Per-Attempt : Observe attempts for a client call, since a call can have multiple attempts due to retry or hedging.
+  * Server : Observe a call received at the server.
+* LB Policy : Observe various load-balancing policies
+  * Weighted Round Robin (experimental)
+  * Pick-First (experimental)
+* XdsClient (experimental)
+
+> [!NOTE]
```

Comment:
Looks like neither of the note callouts are rendering correctly.

Click on the details of the "netlify/grpc-io/deploy-preview" GitHub check to see a preview.

![image](https://github.com/grpc/grpc.io/assets/12451183/8f151766-4a50-4a2a-a32d-87c1e43da53e)

Reply:
I see. We're not using github to display the webpages. Fixed